src/ActionRow/index.jsx

```diff
@@ -1,36 +1,57 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
 const ActionRow = ({
   as,
   isStacked,
+  isStackedReversed,
+  isStackedCentered,
   children,
+  className,
   ...props
-}) => React.createElement(
-  as,
-  {
-    ...props,
-    className: classNames(props.className, {
-      'pgn__action-row': !isStacked,
-      'pgn__action-row-stacked': isStacked,
-    }),
-  },
-  children,
-);
+}) => {
+  const stackedDirection = useMemo(
+    () => {
+      if (!isStacked) {
+        return null;
+      }
+      return isStackedReversed ? 'column-reverse' : 'column';
+    },
+    [isStacked, isStackedReversed],
+  );
+
+  return React.createElement(
+    as,
+    {
+      ...props,
+      className: classNames(className, {
+        'pgn__action-row': !isStacked,
+        'pgn__action-row-stacked': isStacked,
+        [`pgn__action-row-stacked-${stackedDirection}`]: stackedDirection,
+        'pgn__action-row-stacked-centered': isStackedCentered,
+      }),
+    },
+    children,
+  );
+};
 
 ActionRow.propTypes = {
   as: PropTypes.elementType,
   className: PropTypes.string,
   children: PropTypes.node,
   isStacked: PropTypes.bool,
+  isStackedReversed: PropTypes.bool,
+  isStackedCentered: PropTypes.bool,
```

Comment:
Is is better to use separate values: 

```
isStacked
isReversed
justifyContent
```
?

Reply:
No longer relevant 😄